### PR TITLE
Feature/pelagos 3704 add dataset status facet

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/Pelagos/Util/Search.php [hunk 1]
```diff
@@ -235,13 +265,85 @@ private function getFundingOrgInfo(array $aggregations): array
...
+            if (array_key_exists($status, $aggregations)) {
+                return $aggregations[$status];
+            }
+        };
+
+        $statusInfo = [
+            [
+                'id' => 1,
+                'name' => 'Unavailable & Expected Soon',
+                'count' => $datasetCount(DatasetSubmission::AVAILABILITY_STATUS_NOT_AVAILABLE)
```
Comment (mickel1138) [inline src/Pelagos/Util/Search.php]: This count sometimes return null, and not zero. That looks odd on the search page.

File: src/Pelagos/Bundle/AppBundle/Resources/public/js/search.js [hunk 2]
```diff
@@ -4,11 +4,12 @@ $(document).ready(function () {
     "use strict";
     let pageSize = 10;
     let count = $("#count").attr("data-content");
-    let urlParts = window.location.search.split("?");
-    let queryParse = parseQueryString(urlParts[1]);
-    let startPage = `${queryParse.page ? `${queryParse.page}` : 1}`;
-    let rgId = `${queryParse.resGrp}`;
-    let foId = `${queryParse.fundOrg}`;
+    const parsed = queryString.parse(location.search);
+    console.log(parsed);
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/public/js/search.js]: debug

File: src/Pelagos/Util/Search.php (hunk 1 above)
Comment (praneethpr) [reply src/Pelagos/Util/Search.php]: Good catch, returning 0 in the else case now 👍

File: src/Pelagos/Bundle/AppBundle/Resources/public/js/search.js (hunk 2 above)
Comment (praneethpr) [reply src/Pelagos/Bundle/AppBundle/Resources/public/js/search.js]: Removed it

**Pull request merged**